Plots/Trajectory/NCL_polyg_14.py

```diff
@@ -0,0 +1,85 @@
+
+###############################################################################
+# Import packages:
+import numpy as np
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy.feature as cfeature
+from geographiclib.geodesic import Geodesic
+
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+###############################################
+
+def Plot(color, row, col, pos, title, style, pt):
+
+    fig = plt.figure(figsize=(12,12))
+    ax = fig.add_subplot(row, col, pos, projection=ccrs.PlateCarree())
+    
+    ax.set_extent([-125,-60,15,65], ccrs.PlateCarree())
```

Comment:
I recommend having these coordinates as arguments for this function instead of hard coded.